File: Course 2 Project/Backend/e-shopping-app-backend/src/main/java/com/eshopping/service/LoginService.java
package com.eshopping.service;

import com.eshopping.bean.Login;
import com.eshopping.dao.LoginDao;

public class LoginService {
	LoginDao ld = new LoginDao();
	
	public String signIn(Login login) {
			if(ld.signIn(login)) {
					if(login.getTypeofuser().equals("admin")) {
						return "Admin login successfully";
					}else {
						return "Customer login successfully";
					}
			}else {
				return "Failure try once again";
			}
	}
	
	public String signUp(Login login) {
		if(login.getTypeofuser().equals("admin")) {
			return "you can't create admin account";
		}else if(ld.signUp(login)>0) {
			return "Account created successfully";
		}else {
			return "Account didn't create";
		}
	}
}

File: Programs/Angular Project/angular-forms/src/app/login.service.ts

import {Injectable} from '@angular/core';

@Injectable()
export class LoginService {

    checkUser(login:any):string {

        if(login.email=="akash@gmail.com" && login.password=="123"){
            return "successfully login"
        }else {
            return "failure try once again";
        }
    }
}